vendor/github.com/containers/common/pkg/config/containers.conf

```diff
@@ -313,7 +313,7 @@
 # associated with the  pod.  This container does nothing other then sleep,
 # reserving the pods resources for the lifetime of the pod.
 #
-# infra_image = "k8s.gcr.io/pause:3.2"
+# infra_image = "k8s.gcr.io/pause:3.4.1"
```

Comment:
I'm assuming the vendor/ code change was just a global search-replace, but you probably need to revert changes in that tree until https://github.com/kubernetes/kubernetes/pull/98205 lands so that `verify/vendor` passes.

Reply:
Thanks, @TBBle.

We should update dependencies in  https://github.com/containers/common/blob/master/pkg/config/default.go#L48 along with it? I updated the vendor package and will update it later.

Reply:
Ah, yes. Sorry, I don't know why I thought that was vendored from k8s.io. -_- The PR for containers/common makes sense, but as noted there, that config isn't used by this project, so it wouldn't be a blocker for this project.

Reply:
Just do a quick fix. I missed many things.  

Thanks for helpful comments.